src/Services/Implementations/DefaultAlgoliaSearchService.cs

```diff
@@ -144,7 +147,14 @@ public string GetFilterablePropertyName(PropertyInfo property)
 
         public bool IsIndexingEnabled()
         {
-            var existingKey = SettingsKeyInfoProvider.GetSettingsKeyInfo(KEY_INDEXING_ENABLED);
+            var indexingEnabled = ValidationHelper.GetBoolean(appSettingsService[APP_SETTINGS_KEY_INDEXING_ENABLED], true);
+    
+            if (!indexingEnabled)
```

Comment:
Here's my thought process:
- The default integration would be no app setting, which should default to "indexing enabled"
- If someone adds the app setting, it would be to disable indexing via code/configuration even if the db had it enabled. The local setting would override whatever is in the db allowing a developer who is connecting to a shared database to disable the executing of indexing in their local application process.
- If a someone adds a the app setting and sets it to `true`, that would be the same as not having it and the db setting would then be checked.

Reply:
That's a good idea! I'll push that change up.

Reply:
Updated!